### ISSUE TITLE
Pages will not change with the latest stencil-router and @stencil/core@0.0.7

### ISSUE THREAD

**Issue by jgw96** [ionic-team/stencil-router#23]
With the latest stencil-router and stencil/core@0.0.7 certain pages will not be navigated too.

Repro Steps:
- git clone https://github.com/ionic-team/stencil-site
- npm install @stencil/core@0.0.7 && @stencil/router@0.0.17
- navigate to the docs page (docs button in header)
- try navigating to some of the pages in the sidemenu

This is what the docs route looks like:

```
         <stencil-route
              url="/docs/:pageName"
              routeRender={(props: { [key: string]: any }) => {
                const map = {
                  'intro': 'intro/index.html',
                  'browser-support': 'intro/browsers.html',
                  'getting-started': 'start/index.html',
                  'my-first-component': 'basics/my-first-component.html',
                  'templating': 'basics/templating.html',
                  'decorators': 'basics/decorators.html',
                  'events': 'basics/events.html',
                  'component-lifecycle': 'basics/component-lifecycle.html',
                  'stencil-config': 'basics/stencil-config.html',
                  'forms': 'basics/forms.html',
                  'handling-arrays': 'basics/handling-arrays.html',
                  'testing': 'basics/testing.html',
                  'config': 'compiler/config.html',
                  'server-side-rendering': 'advanced/ssr/index.html',
                  'routing': 'addons/stencil-router.html',
                  'service-workers': 'advanced/service-worker/index.html',
                  'distribution': 'advanced/distribution/index.html',
                  'webpack': 'advanced/webpack/index.html',
                  'prerendering': 'advanced/pre-rendering/index.html',
                  'shadow-dom': 'advanced/shadow-dom/index.html'
                };
                return (
                  <document-component pages={[map[props.match.params.pageName]]} />
                );
              }}
            />
```

**Comment by jthoms1** [comment]
Does this throw an error or just do nothing?

**Comment by hufan-akari** [comment]
https://github.com/ionic-team/stencil/issues/312
https://github.com/hufan-akari/stencil-site/commit/cae43b51159dd865ad3fbcd9662f0b7c74b4f640

**Comment by jthoms1** [comment]
This was an issue with stencil and was resolved in commit eedad6b3d689d0212ea9b5d3b0fc3c3a472ecbb6.
